client/apps/admin/components/application_instances/form.jsx

```diff
@@ -6,6 +6,14 @@ import Input from '../common/input';
 import Textarea from '../common/textarea';
 import Warning from '../common/warning';
 
+function prettyJSON(str) {
```

Comment:
This feels like a function that could probably live in a utils file somewhere instead

Reply:
Probably but I didn't want to make a bunch of changes to the admin tool.